catalog/aws.go

```diff
@@ -26,17 +26,11 @@ const (
 	ConsulAWSID     = "external-aws-id"
 )
 
-type namespace struct {
-	id     string
-	name   string
-	isHTTP bool
-}
```

Comment:
This object was used to represent the data associated with an AWS Service discovery namespace. Rather than continuing to copy and parse the data, I just inlined it. The reason why is below.